src/lndAuthTypes.d.ts

```diff
@@ -0,0 +1,16 @@
+type nodeType = "offchain" | "onchain"
+
+interface ILndParams {
```

Comment:
I know you're just moving code around... but since you're touching it we may as well improve it. I think pure data types that have no behaviour don't need to be interfaces in general. WDYT?

Reply:
I think in this case I chose `interface` so that I could extend it later on (`ILndParamsAuthed` extends from `ILndParams`). Extending felt more correct than declaring it as an " intersection type" (by using `&` to join them), and it felt odd to declare `LndParams` as a type and then extend `ILndParamsAuthed` from it as an interface, so I chose to leave them both as interfaces.

That being said, it could be useful to come up with a general set of expectations around when to use types vs. interfaces. Right off the bat I saw that once a type is created it can't be edited whereas new properties can always be added to an interface. Also interfaces can extend from both types and interfaces whereas types can't extend from anything.

Reply:
> For e.g., we can reserve interface only for things that we mean to implement?

Yes this is the approach I would take.